frappe_graphql/api.py

```diff
@@ -0,0 +1,121 @@
+from graphql import GraphQLError
+from typing import List
+
+import frappe
+from .graphql import execute
+
+from .utils.http import get_masked_variables, get_operation_name
+
+
+@frappe.whitelist(allow_guest=True)
+def execute_gql_query():
+    query, variables, operation_name = get_query()
+    output = execute(
+        query=query,
+        variables=variables,
+        operation_name=operation_name
+    )
+
+    frappe.clear_messages()
+    frappe.local.response = output
```

Comment:
@faztp12 What's the difference between returning output vs setting it to local response like here?

Reply:
@assemmarwan when output is returned, it will be wrapped in `message`

Reply:
Thanks!